src/map/entities/battleentity.cpp

```diff
@@ -878,9 +878,7 @@ void CBattleEntity::SetSLevel(uint8 slvl)
     TracyZoneScoped;
     if (!settings::get<bool>("map.INCLUDE_MOB_SJ") && this->objtype == TYPE_MOB && this->objtype != TYPE_PET)
     {
-        // Technically, we shouldn't be assuming mobs even have a ratio they must adhere to.
-        // But there is no place in the DB to set subLV right now.
-        m_slvl = (slvl > (m_mlvl >> 1) ? (m_mlvl == 1 ? 1 : (m_mlvl >> 1)) : slvl);
+        m_slvl = m_mlvl; // All mobs have a 1:1 ratio of MainJob/Subjob
```

Comment:
curious how this will affect Maat - right now he has no subjob.

Reply:
I did a Maat revamp here after the stat audit: https://github.com/AirSkyBoat/AirSkyBoat/pull/2303 I can try to find the notes we used for the FTS/stat ranks. I should have it somewhere.